docs/docs/quickstart.mdx

```diff
@@ -127,40 +130,13 @@ SMTP server the demo uses to show how e.g. email verification works.
 
 ### Clone ORY Kratos and run it in Docker
 
-To get this example working, you will need Git and
-[Docker, and Docker Compose](https://docs.docker.com/get-docker/) installed on
-your system. No other dependencies are required. Before you start, make sure
-that Docker has enough disk space.
-
-:::tip
-
-This tutorial uses Docker-Compose volumes which have reported to run out of disk
-space. Check the remaining disk space using `docker system df`. If the volumes
-are above the 85% threshold,
-[prune old Docker objects](https://docs.docker.com/config/pruning/) before you
-start!
-
-:::
-
-:::tip
-
-If you encounter build errors (e.g. network timeout), make sure that the network
-is running correctly and run `macke docker` again. If the problem persists, feel
-free to [open an issue](https://github.com/ory/kratos/issues/new/choose).
-
-:::
+To get this example working, you will need Git,
+[Docker](https://docs.docker.com/get-docker/), and [Docker Compose](https://docs.docker.com/compose/install/) installed on
+your system. No other dependencies are required.
 
 Let's clone ORY Kratos and run `docker-compose`:
 
 ```shell script
-git clone https://github.com/ory/kratos.git
-# or if you have git+ssh set up:
-#  git clone git@github.com:ory/kratos.git
-cd kratos
-git checkout v0.3.0-alpha.1
-make quickstart
-
-# or if you don't have make installed:
```

Comment:
Please revert this. It's very unclear what needs to be done without these instructions.

Reply:
I'm a bit unclear on this part. Why provide instructions for `make` here at all? Docker's commands require only the necessary software (Docker and Docker Compose) so using `make` seemed a little tangential to just getting users up and running quickly, and I think if anything it obfuscates what is actually going on with Docker.

The plan Docker instructions underneath this, in my opinion, make it much clearer what's happening.

Reply:
Ok, that's a good point! Let's remove the `make quickstart` then. But we still need to check out the GitHub repository, otherwise all the config files will be missing!

Reply:
Ah, that makes sense! Fixed.